### PR TITLE
Assignment Details page - New UI

### PR DESCRIPTION
**Pull request by aayush-05**
image

### CODE REVIEW

File: app/views/assignments/show.html.erb [hunk 1]
```diff
@@ -1,140 +1,103 @@
...
-        <div class="list-group" id="list-tab" role="tablist" id="submissionTable" style="max-height:600px;overflow:hidden;overflow-y:scroll">
-          <% @assignment.project_order.each do |project| %>
-              <a class="list-group-item list-group-item-action" id=<%= project.id%> data-grade="<%= project.grade_str %>" data-remarks="<%= project.remarks_str %>"><%= project.author.name  %></a>
-          <%end%>
-
-        </div>
+    <div class="col-3 col-3-submission-list">
+      <div class="list-group" id="list-tab" role="tablist" id="submission-table">
+        <% @assignment.project_order.each do |project| %>
+          <a class="list-group-item list-submission" id=<%= project.id %> data-grade="<%= project.grade_str %>" data-remarks="<%= project.remarks_str %>"><%= project.author.name %></a>
```
[the remark this answers is not in the document]
Comment (aayush-05) [reply app/views/assignments/show.html.erb]: No idea about this, even tried autoformatting.

**Comment by tachyons** [comment]
Ci is failing

**Comment by aayush-05** [comment]
Done @tachyons . Also can this be pushed to the beta deployment. I just want to confirm with the UI, when some other user submits an assignment (hard to accomplish on local server)

**Comment by aayush-05** [comment]
@nitin10s please check.

**Comment by Ananthavijay** [comment]
````Edit```` and ````Export````should have the same class. Because after the deadline of the assignment is over, the ````Edit```` button will be changed to ````Export```` and ````Back```` button should be rightmost ( next to Export code )

**Comment by satu0king** [comment]
I see that you have done smart time setting logic for the deadline, but there seems to be some issue, can you tell me the logic used?

**Comment by satu0king** [comment]
I see that the deadline logic has been merged in another PR, it needs to be reviewed.

**Pull request merged**